src/backend/assets/js/tabs.js

```diff
@@ -26,6 +30,19 @@ tabs = function(options) {
       goToTab(index);
     });
   };
+  handleDisplay = function(className) {
+    className = className.split(' ')[0];
+    var dates = document.getElementsByClassName('date-filter');
+    for(i =0; i< dates.length; i++) {
+      if(dates[i].classList.contains(className)) {
```

Comment:
Why should we write so much JS DOM optimization? Are we not using jQuery. Handling DOM with jQuery will be fast,
Please compare the performance  `document.getElementsByClassName` and `$('.classname')` using console.time()